clusters/istiomesh.yaml

```diff
@@ -0,0 +1,59 @@
+concourse-team: gsp
+concourse-username: gsp
+concourse-url: https://cd.gds-reliability.engineering
+concourse-pipeline-name: istiomesh-deployer
+account-id: "011571571136"
+account-name: "sandbox"
+account-role-arn: "arn:aws:iam::011571571136:role/deployer"
+cluster-name: "istiomesh"
+cluster-number: "101"
+cluster-domain: "london.istiomesh.govsvc.uk"
+splunk-enabled: "0"
+splunk-hec-url: "NOTAURL"
+k8s-splunk-hec-token: "NOTATOKEN"
+k8s-splunk-index: "NOTANINDEX"
+hsm-splunk-hec-token: "NOTATOKEN"
+hsm-splunk-index: "NOTANINDEX"
+vpc-flow-log-splunk-hec-token: "NOTATOKEN"
+vpc-flow-log-splunk-index: "NOTANINDEX"
+eks-version: "1.12"
+platform-uri: "https://github.com/alphagov/gsp.git"
+platform-organization: "alphagov"
+platform-repository: "gsp"
+platform-version: "proxy-node-mesh"
+platform-pre-release: true
+config-uri: "https://github.com/alphagov/tech-ops-cluster-config.git"
+config-organization: "alphagov"
+config-repository: "tech-ops-cluster-config"
+config-version: "proxy-node-mesh"
+config-path: "clusters/istiomesh.yaml"
+config-values-path: "clusters/istiomesh-values.yaml"
+users-uri: "git@github.com:alphagov/gds-trusted-developers.git"
+users-organization: "alphagov"
+users-repository: "gds-trusted-developers"
+users-version: "proxy-node-mesh"
+users-path: "users"
+disable-destroy: false
+worker-instance-type: m5d.large
+worker-count: 3
+ci-worker-instance-type: t3.small
+ci-worker-count: 3
+github-approval-count: 0
+github-client-secret: "NOTASECRET"
+github-client-id: "NOTID"
+github-approvers: ["blairboy362"]
```

Comment:
I don't know what this does but does it need any more names here? `samcrang` maybe?

Reply:
With an approval count of "0" for this cluster I think this is basically irrelevant, but it got upset when being given an empty list and investigating / fixing that was not in scope.